Tests/SwiftGRPCNIOTests/NIOServerTests.swift

```diff
@@ -92,211 +94,106 @@ class NIOServerTests: NIOServerTestCase {
     ]
   }
 
-  static let lotsOfStrings = (0..<1000).map { String(describing: $0) }
-
-  var eventLoopGroup: MultiThreadedEventLoopGroup!
-  var server: GRPCServer!
-
-  override func setUp() {
-    super.setUp()
-
-    // This is how a GRPC server would actually be set up.
-    eventLoopGroup = MultiThreadedEventLoopGroup(numberOfThreads: 1)
-    server = try! GRPCServer.start(
-      hostname: "localhost", port: 5050, eventLoopGroup: eventLoopGroup, serviceProviders: [EchoProvider_NIO()])
-      .wait()
-  }
-
-  override func tearDown() {
-    XCTAssertNoThrow(try server.close().wait())
-
-    XCTAssertNoThrow(try eventLoopGroup.syncShutdownGracefully())
-    eventLoopGroup = nil
-
-    super.tearDown()
-  }
+  static let aFewStrings = ["foo", "bar", "baz"]
+  static let lotsOfStrings = (0..<10_000).map { String(describing: $0) }
 }
 
 extension NIOServerTests {
-  func testUnary() {
-    XCTAssertEqual("Swift echo get: foo", try! client.get(Echo_EchoRequest(text: "foo")).text)
+  func testUnary() throws {
+    let options = CallOptions(timeout: nil)
```

Comment:
Why this special option here?

In general, I would always specify a timeout in tests, to avoid stalling tests indefinitely in case the request never finishes for whatever reason. How about having a test-local `defaultOptions` variable specifying a 5-second timeout, and using that in all tests?

Reply:
> Why this special option here?

It's not meant to be!

> In general, I would always specify a timeout in tests, to avoid stalling tests indefinitely in case the request never finishes for whatever reason. How about having a test-local `defaultOptions` variable specifying a 5-second timeout, and using that in all tests?

Done.